tutorials/relative_time_constraints/index.md

```diff
@@ -0,0 +1,629 @@
+---
+title:  Dating with relative constraints
+subtitle: Constraining the relative order of nodes when dating a phylogeny
+authors: Dominik Schrempf, Gergely Szöllősi, and Bastien Boussau
+level: 3
+prerequisites:
+- intro
+- mcmc
+- ctmc
+index: true
+---
+
+Introduction
+--
+{:.section}
+
+In phylogenetics, the term __dating__ denotes the inference of node ages of
+phylogenies. For example, dating a species tree involves inference of the times
+when the species split. Dating phylogenies is central to understanding the
+evolution of life on Earth. However, evolutionary sequences evolve with
+different rates, an observation that has been termed __relaxed molecular
+clock__. In general, it is difficult to map a phylogeny obtained from a multiple
+sequence alignment, and with branch lengths measured in average number of
+substitutions per site, to a phylogeny with branch lengths measured in actual
+units of time. For this reason, the molecular clocks are calibrated with
+information gained from __fossils__, which can be accurately dated.
+Incorporating fossil information enables __calibration__ of node ages of a
+phylogeny.
+
+In this tutorial, we explore another possibility to improve on the accuracy of
+dating a phylogeny. Namely, horizontal gene transfers and ancient evolutionary
+relationships such as symbioses are informative about the order of nodes of a
+phylogeny of interest. That is, __horizontal gene transfers__ define __relative
+node constraints__. Relative node constraints can be particularly helpful when
+the geological record is sparse, for example, for microorganisms, which
+represent the vast majority of extant and extinct biodiversity. Combination of
+node calibrations with relative node constraints can significantly improve both
+accuracy and resolution of molecular clock estimates {% cite Szollosi2020 %}.
```

Comment:
Some statement of what this tutorial hopes to convey or teach would be good. Something like
"In this tutorial we will explain how use relative node constraints to..."

Reply:
I think the message of this paragraph is very clear. Calibrations and constraints improve the estimated node ages. Here we use both. If you think it is not  clear enough, please feel free to change the paragraph!